### PR TITLE
aksRightSizing

### PR DESCRIPTION
**Pull request by oded7hoffman**

### CODE REVIEW

File: api/services/ocean/aks/paths/rightSizing.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+post:
+  summary: "Fetch Right Sizing Recommendations"
```
Comment (EladTanaami) [inline api/services/ocean/aks/paths/rightSizing.yaml:2]: Is that a post or a get route?

File: api/services/ocean/aks/responses/rightSizingResponse.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+description: Ocean RightSizing Recommendation Response
```
Comment (EladTanaami) [inline api/services/ocean/aks/responses/rightSizingResponse.yaml:1]: The file name is not by the convension need to be like rightSizingCreate

File: api/services/ocean/aks/paths/rightSizing.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+post:
```
Comment (EladTanaami) [inline api/services/ocean/aks/paths/rightSizing.yaml:1]: Why didnt update the spot.yaml file?

File: api/services/ocean/aks/paths/rightSizing.yaml (hunk 1 above)
Comment (oded7hoffman) [reply api/services/ocean/aks/paths/rightSizing.yaml:2]: It is a POST route

File: api/services/ocean/aks/paths/rightSizing.yaml (hunk 3 above)
Comment (oded7hoffman) [reply api/services/ocean/aks/paths/rightSizing.yaml:1]: Added to spot.yaml

File: api/services/ocean/aks/responses/rightSizingResponse.yaml (hunk 2 above)
Comment (oded7hoffman) [reply api/services/ocean/aks/responses/rightSizingResponse.yaml]: Modified to rightSizingSearch

File: api/services/ocean/aks/paths/rightSizing.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  summary: "Fetch Right Sizing Recommendations"
+  description: >
+    Get right-sizing recommendations for an Ocean cluster and filter them according to namespace or label.
+  operationId: "oceanAzureFilterRightSizingWithFilter"
+  tags:
+    - "Ocean AZURE"
```
Comment (EladTanaami) [inline api/services/ocean/aks/paths/rightSizing.yaml:7]: Wrong Tag name: need to be 'Ocean AKS' -

Comment (oded7hoffman) [reply api/services/ocean/aks/paths/rightSizing.yaml]: Fixed

**Review by EladTanaami — APPROVED** [review]

**Pull request merged**